src/main/java/com/amplitude/Event.java

```diff
@@ -151,7 +151,7 @@
    * want to associate events with a particular system. A session_id of -1 is the same as no
    * session_id specified.
    */
-  public long sessionId;
+  public long sessionId = -1;
```

Comment:
looks good, can we add a quick test to assert the default value?

Reply:
It's been done